src/data/imprint/types.ts

```diff
@@ -10,8 +10,8 @@ export interface Imprint {
 
 export type ImprintPageOnDisk = {
   id: string
-  head: TranslationCollection<Head>
-} & TranslationCollection<{
+  head: HeadLocalized
+} & TranslationCollectionOld<{
```

Comment:
Why use the old one here?

Reply:
Because the content is markdown and we don't have a localizedMarkdown widget.

Reply:
Ok